### PR TITLE
Edit profile - form submission

### PR DESCRIPTION
**Pull request by mmmavis**
Two ways to go the the edit profile page (make sure you are logged in first. followup PR will be filed to take care https://github.com/mozilla/network-pulse/issues/685)
- visit `/profile/me` and click on "edit profile" link under your profile pic
- visit `/myprofile` directly

Things to verify

- form is prefilled with user's current profile data
- you are able to update your profile
- form validation works

### CODE REVIEW

File: js/form-validator.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import url from 'url';
```
Comment (mmmavis) [inline js/form-validator.js]: This is all existing code but I moved it to here (`/js/form-validator.js`). In a followup PR I'll remove `/pages/add/form/validator.js` and make the Add page uses this js file instead.

File: pages/profile-edit/form/fields.js [hunk 2]
```diff
@@ -1,84 +1,80 @@
 import React from 'react';
 import { Link } from 'react-router';
+import validator from '../../../js/form-validator';
 import IssuesField from '../../../components/form-fields/issues.jsx';
-import HelpTypesField from '../../../components/form-fields/help-types.jsx';
```
Comment (mmmavis) [inline pages/profile-edit/form/fields.js]: removed a few non-MVP fields

File: pages/profile-edit/profile-edit.jsx [hunk 3]
```diff
@@ -5,20 +5,29 @@ import { Form } from 'react-formbuilder';
 import HintMessage from '../../components/hint-message/hint-message.jsx';
```
Comment (mmmavis) [inline pages/profile-edit/profile-edit.jsx]: Most of the code and logic are very similar to what we have for the "add new entry" page. The part that's different is the profile edit form should be prefilled with user's current profile data (which is done via `loadCurrentProfile()`.

**Comment by mmmavis** [comment]
@alanmoo this is ready for review! See top comment for things to check.

**Comment by xmatthewx** [comment]
Intro copy and a couple other minor tweaks for legal are tracked here in #703.

**Comment by mmmavis** [comment]
Merging this in nowww 🎉

**Pull request merged**